nncf/common/logging/track_progress.py

```diff
@@ -47,6 +49,73 @@ def render(self, task: Task) -> Text:
         return Text("•")
 
 
+class TimeElapsedColumnWithStyle(ProgressColumn):
+    """
+    Renders time elapsed.
+
+    Similar to TimeElapsedColumn, but with addition of style parameter.
+    """
+
+    def __init__(self, style: Union[str, StyleType]):
+        super().__init__()
+        self.style = style
+
+    def render(self, task: "Task") -> Text:
+        """Show time elapsed."""
+        elapsed = task.finished_time if task.finished else task.elapsed
+        if elapsed is None:
+            return Text("-:--:--", style=self.style)
+        delta = timedelta(seconds=max(0, int(elapsed)))
+        return Text(str(delta), style=self.style)
+
+
+class TimeRemainingColumnWithStyle(ProgressColumn):
```

Comment:
Is there a way to override default styles?
Maybe it would be better to inherit from `TimeRemainingColumn`?
```python
class TimeRemainingColumnWithStyle(TimeRemainingColumn):

    def render(self, task: "Task") -> Text:
        text = super().render(task)
        return Text(text._text[0], style=INTEL_BLUE_COLOR_STYLE)
```
Ofc it's access to protected attribute but looks shorter and without copypast.

Reply:
Haven't found a way to override default style. Changed custom columns to the way you suggested.